package.json

```diff
@@ -291,6 +293,12 @@
     "enhanced-resolve": "5.8.3",
     "enzyme": "3.11.0",
     "enzyme-adapter-react-16": "1.15.6",
+    "eslint": "7.32.0",
+    "eslint-plugin-import": "2.25.2",
+    "eslint-plugin-node": "11.1.0",
+    "eslint-plugin-promise": "5.1.1",
+    "eslint-config-standard-with-typescript": "21.0.1",
+    "eslint-plugin-jest": "25.2.2",
```

Comment:
the new deps pass the audit-deps command, which is good: https://ci.brave.com/blue/organizations/jenkins/pr-brave-browser-tslint_to_eslint-noplatform/detail/pr-brave-browser-tslint_to_eslint-noplatform/9/pipeline/116

however if possible i'd like to wait until @petemill finishes un-ignoring these advisories to make sure that these new deps don't have these vulnerabilities:

* https://github.com/advisories/GHSA-c36v-fmgq-m8hx
* https://github.com/advisories/GHSA-4jqc-8m5r-9rpr

Reply:
I have confirmed these deps do not have those vulnerabilities. I confirmed by adding these deps to the branch at https://github.com/brave/brave-core/pull/10993 which unignores those vulnerabilities, and running `audit_deps` again locally. The vulnerability errors did not come back.